librz/core/cmd/cmd_print.c

```diff
@@ -5418,399 +5188,6 @@ RZ_IPI int rz_cmd_print(void *data, const char *input) {
-							rz_cons_printf("%s\n", pj_string(pj));
-							pj_free(pj);
-						} else {
-							core->num->value = rz_core_print_disasm(
-								core->print, core, b->addr, block,
-								b->size, 9999, 0, 2, input[2] == 'J', NULL, NULL);
-						}
-						free(block);
-						pd_result = 0;
-					}
-				} else {
-					eprintf("Cannot find function at 0x%08" PFMT64x "\n", core->offset);
-					core->num->value = 0;
-				}
-			}
-			break;
-		case 's': // "pds" and "pdsf"
-			processed_cmd = true;
-			if (input[2] == '?') {
-				rz_core_cmd_help(core, help_msg_pds);
-			} else {
-				disasm_strings(core, input, NULL);
-			}
-			break;
-		case 'f': // "pdf"
-			processed_cmd = true;
-			if (input[2] == '?') {
-				rz_core_cmd_help(core, help_msg_pdf);
-			} else if (input[2] == 's') { // "pdfs"
-				ut64 oseek = core->offset;
-				int oblock = core->blocksize;
-				RzAnalysisFunction *f = rz_analysis_get_fcn_in(core->analysis, core->offset,
-					RZ_ANALYSIS_FCN_TYPE_FCN | RZ_ANALYSIS_FCN_TYPE_SYM);
-				if (f) {
-					ut32 rs = rz_analysis_function_realsize(f);
-					ut32 fs = rz_analysis_function_linear_size(f);
-					rz_core_seek(core, oseek, SEEK_SET);
-					rz_core_block_size(core, RZ_MAX(rs, fs));
-					disasm_strings(core, input, f);
-					rz_core_block_size(core, oblock);
-					rz_core_seek(core, oseek, SEEK_SET);
-				}
-				processed_cmd = true;
-			} else {
-				ut32 bsz = core->blocksize;
-				RzAnalysisFunction *f = rz_analysis_get_fcn_in(core->analysis, core->offset, RZ_ANALYSIS_FCN_TYPE_ROOT);
-				if (!f) {
-					f = rz_analysis_get_fcn_in(core->analysis, core->offset, 0);
-				}
-				if (f && input[2] == 'j') { // "pdfj"
-					pj = pj_new();
-					if (!pj) {
-						break;
-					}
-					rz_core_print_function_disasm_json(core, f, pj);
-					rz_cons_printf("%s\n", pj_string(pj));
-					pj_free(pj);
-					pd_result = 0;
-				} else if (f) {
-					ut64 linearsz = rz_analysis_function_linear_size(f);
-					ut64 realsz = rz_analysis_function_realsize(f);
-					if (realsz + 4096 < linearsz) {
-						eprintf("Linear size differs too much from the bbsum, please use pdr instead.\n");
-					} else {
-						ut64 start = f->addr; // For pdf, start disassembling at the entrypoint
-						ut64 end = rz_analysis_function_max_addr(f);
-						if (end > start) {
-							ut64 sz = end - start;
-							ut8 *buf = calloc(sz, 1);
-							if (buf) {
-								(void)rz_io_read_at(core->io, start, buf, sz);
-								core->num->value = rz_core_print_disasm(core->print, core, start, buf, sz, sz, 0, 1, 0, NULL, f);
-								free(buf);
-							}
-						}
-					}
-					pd_result = 0;
-				} else {
-					eprintf("pdf: Cannot find function at 0x%08" PFMT64x "\n", core->offset);
-					processed_cmd = true;
-					core->num->value = 0;
-				}
-				if (bsz != core->blocksize) {
-					rz_core_block_size(core, bsz);
-				}
-			}
-			l = 0;
-			break;
-		case 'p': // "pdp"
-			processed_cmd = true;
-			disasm_ropchain(core, core->offset, 'D');
-			pd_result = true;
-			break;
```

Comment:
is this moved elsewhere?

Reply:
Yes, it's handled by `rz_cmd_disassemble_ropchain_handler` now.